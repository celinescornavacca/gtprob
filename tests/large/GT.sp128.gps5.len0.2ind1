((((((((((QC3,QC4),((QC1,QC2),QC0)),((RC3,RC4),(RC0,(RC1,RC2)))),(PC0,(((PC3,PC4),PC2),PC1))),(QB0,(QB1,((QB3,QB4),QB2)))),(((YA3,YA4),(YA0,(YA1,YA2))),(((ZA0,(ZA3,ZA4)),(ZA1,ZA2)),((AB0,(AB1,AB2)),(AB3,AB4))))),(((IA1,IA2),(IA3,IA4)),IA0)),(((((VB1,((VB3,VB4),VB2)),VB0),((WB0,WB1),(WB2,(WB3,WB4)))),((((QA0,((QA2,(QA3,QA4)),QA1)),((RA3,RA4),(RA0,(RA1,RA2)))),(((((XA3,XA4),(XA0,(XA1,XA2))),((((UD0,(UD3,UD4)),(UD1,UD2)),(VD0,(VD1,((VD3,VD4),VD2)))),((((DD3,DD4),DD2),(DD0,DD1)),(ED0,(((ED3,ED4),ED2),ED1))))),(OA0,((OA3,OA4),(OA1,OA2)))),(((((CD2,(CD3,CD4)),CD1),CD0),((((MD1,MD2),MD0),(MD3,MD4)),((ND0,ND1),((ND3,ND4),ND2)))),(((DC1,DC2),(DC0,(DC3,DC4))),((EC3,EC4),(EC0,(EC1,EC2))))))),(((NE0,NE1),(NE2,(NE3,NE4))),(OE0,(OE1,((OE3,OE4),OE2)))))),((((JA3,JA4),JA0),(JA1,JA2)),((((BB3,BB4),(BB0,(BB1,BB2))),((((OB3,OB4),OB0),(OB1,OB2)),((((TE3,TE4),((TE1,TE2),TE0)),((((UE3,UE4),UE2),(UE0,UE1)),(VE0,(VE1,(VE2,(VE3,VE4)))))),((CC0,CC1),(CC2,(CC3,CC4)))))),((KA0,KA1),((KA3,KA4),KA2)))))),((((((HA3,HA4),(HA0,(HA1,HA2))),((((((JD3,JD4),JD0),(JD1,JD2)),(((KD1,KD2),KD0),(KD3,KD4))),((ZB0,(ZB1,((ZB3,ZB4),ZB2))),((AC0,AC1),(AC2,(AC3,AC4))))),((FD0,((FD1,FD2),(FD3,FD4))),((FE0,(((FE3,FE4),FE2),FE1)),(GE0,((GE2,(GE3,GE4)),GE1)))))),(((((QD0,QD1),((QD3,QD4),QD2)),(((RD3,RD4),(RD1,RD2)),RD0)),((XC0,XC1),((XC3,XC4),XC2))),((BC1,((BC3,BC4),BC2)),BC0))),((YC0,(YC1,((YC3,YC4),YC2))),((ZC3,ZC4),(ZC0,(ZC1,ZC2))))),((((WD0,(((WD3,WD4),WD2),WD1)),((XD3,XD4),((XD1,XD2),XD0))),((WA2,(WA3,WA4)),(WA0,WA1))),((((RB2,(RB3,RB4)),(RB0,RB1)),((SB3,SB4),((SB1,SB2),SB0))),((((((GD0,(GD1,(GD2,(GD3,GD4)))),((AE0,((AE2,(AE3,AE4)),AE1)),(((BE3,BE4),BE2),(BE0,BE1)))),(((OC3,OC4),OC0),(OC1,OC2))),(((IC2,(IC3,IC4)),IC1),IC0)),((((YD1,YD2),((YD3,YD4),YD0)),((((ZD3,ZD4),ZD2),ZD1),ZD0)),(((FC1,(FC2,(FC3,FC4))),FC0),(((GC2,(GC3,GC4)),GC1),GC0)))),(((CA2,(CA3,CA4)),CA1),CA0)))))),((((((SD2,(SD3,SD4)),(SD0,SD1)),((TD1,((TD3,TD4),TD2)),TD0)),(((KB1,KB2),((KB3,KB4),KB0)),((LB0,(LB1,LB2)),(LB3,LB4)))),(((FA0,(FA1,FA2)),(FA3,FA4)),((GA3,GA4),(GA0,(GA1,GA2))))),(((((((((UA2,(UA3,UA4)),UA1),UA0),(((CE0,((CE1,CE2),(CE3,CE4))),((DE1,DE2),(DE0,(DE3,DE4)))),(((TB0,((TB2,(TB3,TB4)),TB1)),((((UB3,UB4),UB2),UB1),UB0)),((IB0,(IB3,IB4)),(IB1,IB2))))),(((((JE1,((JE3,JE4),JE2)),JE0),(KE0,((KE2,(KE3,KE4)),KE1))),(((EE3,EE4),EE2),(EE0,EE1))),((((((((VC3,VC4),(VC1,VC2)),VC0),(WC0,(WC1,(WC2,(WC3,WC4))))),((HC0,(HC3,HC4)),(HC1,HC2))),(((NC0,((NC1,NC2),(NC3,NC4))),((((((PE1,PE2),(PE3,PE4)),PE0),(QE0,((QE3,QE4),(QE1,QE2)))),((SC1,((SC3,SC4),SC2)),SC0)),(((((HE3,HE4),(HE1,HE2)),HE0),((IE3,IE4),(IE0,(IE1,IE2)))),(((AD0,(AD3,AD4)),(AD1,AD2)),((BD1,(BD2,(BD3,BD4))),BD0))))),(((MB1,MB2),MB0),(MB3,MB4)))),(((HB0,(HB1,HB2)),(HB3,HB4)),(((TC2,(TC3,TC4)),(TC0,TC1)),(((UC1,UC2),UC0),(UC3,UC4))))),((MA1,MA2),((MA3,MA4),MA0))))),((((((JC1,JC2),(JC0,(JC3,JC4))),(KC0,((KC3,KC4),(KC1,KC2)))),(((EB1,EB2),(EB0,(EB3,EB4))),((FB0,FB1),((FB3,FB4),FB2)))),((((LD2,(LD3,LD4)),LD1),LD0),(((LE0,LE1),(LE2,(LE3,LE4))),(((ME3,ME4),ME0),(ME1,ME2))))),(((LA3,LA4),(LA1,LA2)),LA0))),(EA0,(((EA3,EA4),EA2),EA1))),((((VA0,(VA1,(VA2,(VA3,VA4)))),((((WE3,WE4),(WE0,(WE1,WE2))),((((XE3,XE4),XE2),XE1),XE0)),((JB1,JB2),((JB3,JB4),JB0)))),((DA0,(DA1,((DA3,DA4),DA2))),(((((TA3,TA4),TA2),TA1),TA0),((CB0,((CB1,CB2),(CB3,CB4))),((((((GB3,GB4),GB0),(GB1,GB2)),((((PB0,(PB3,PB4)),(PB1,PB2)),(((HD0,(HD3,HD4)),(HD1,HD2)),(ID0,(ID1,((ID3,ID4),ID2))))),((NB1,NB2),(NB0,(NB3,NB4))))),(((XB1,XB2),(XB0,(XB3,XB4))),(YB0,((YB1,YB2),(YB3,YB4))))),(((DB1,DB2),DB0),(DB3,DB4))))))),((BA1,BA2),((BA3,BA4),BA0)))),(((((((PA3,PA4),PA2),PA1),PA0),(((LC0,(LC3,LC4)),(LC1,LC2)),((MC0,(MC1,MC2)),(MC3,MC4)))),((((OD0,((OD1,OD2),(OD3,OD4))),(((PD3,PD4),(PD1,PD2)),PD0)),((((RE0,(RE3,RE4)),(RE1,RE2)),(SE0,((SE3,SE4),(SE1,SE2)))),(SA0,((SA3,SA4),(SA1,SA2))))),((NA0,NA1),((NA3,NA4),NA2)))),(((AA1,AA2),AA0),(AA3,AA4))))));
