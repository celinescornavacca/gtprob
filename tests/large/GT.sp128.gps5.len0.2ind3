((((AA1,AA2),(AA0,(AA3,AA4))),(((((LB0,(((LB3,LB4),LB2),LB1)),((((((ND3,ND4),ND2),ND1),ND0),(((OD3,OD4),OD0),(OD1,OD2))),(LC0,((LC3,LC4),(LC1,LC2))))),((KA0,KA1),(KA2,(KA3,KA4)))),(((((OA0,(OA1,OA2)),(OA3,OA4)),(((LD0,LD1),(LD2,(LD3,LD4))),(((MD3,MD4),(MD1,MD2)),MD0))),(((((((PB2,(PB3,PB4)),PB1),PB0),((QB1,QB2),(QB0,(QB3,QB4)))),(((((AC1,AC2),(AC3,AC4)),AC0),(((BC1,BC2),BC0),(BC3,BC4))),(HB0,((HB2,(HB3,HB4)),HB1)))),((((CC3,CC4),CC2),(CC0,CC1)),((DC2,(DC3,DC4)),(DC0,DC1)))),(((((BD1,BD2),BD0),(BD3,BD4)),((CD1,CD2),(CD0,(CD3,CD4)))),(((WE0,(((WE3,WE4),WE2),WE1)),(((XE2,(XE3,XE4)),XE1),XE0)),(CB0,((CB1,CB2),(CB3,CB4))))))),(((((((KE0,(((KE3,KE4),KE2),KE1)),((LE0,(LE1,LE2)),(LE3,LE4))),((UC1,UC2),((UC3,UC4),UC0))),(((XA1,XA2),XA0),(XA3,XA4))),(((TA0,((TA3,TA4),(TA1,TA2))),(((((QC2,(QC3,QC4)),QC1),QC0),((RC0,RC1),(RC2,(RC3,RC4)))),(AB0,((AB3,AB4),(AB1,AB2))))),((((PA2,(PA3,PA4)),(PA0,PA1)),(((QA3,QA4),(QA1,QA2)),QA0)),((MA0,(MA1,MA2)),(MA3,MA4))))),(((FB0,(FB3,FB4)),(FB1,FB2)),(GB0,(((GB3,GB4),GB2),GB1)))),((((((GE1,(GE2,(GE3,GE4))),GE0),(((HE3,HE4),HE0),(HE1,HE2))),(((OB0,OB1),((OB3,OB4),OB2)),(((EE0,(EE3,EE4)),(EE1,EE2)),(((FE3,FE4),(FE1,FE2)),FE0)))),((((EA3,EA4),EA2),(EA0,EA1)),((FA0,(FA1,FA2)),(FA3,FA4)))),((JA0,(JA1,(JA2,(JA3,JA4)))),(((((SA1,SA2),((SA3,SA4),SA0)),(((UE2,(UE3,UE4)),(UE0,UE1)),(VE0,(((VE3,VE4),VE2),VE1)))),((((((XD0,(XD3,XD4)),(XD1,XD2)),(((YD2,(YD3,YD4)),YD1),YD0)),(SC0,(SC1,((SC3,SC4),SC2)))),((JD0,((JD2,(JD3,JD4)),JD1)),((KD2,(KD3,KD4)),(KD0,KD1)))),((((IE0,(IE1,IE2)),(IE3,IE4)),((JE1,((JE3,JE4),JE2)),JE0)),((DE3,DE4),(DE0,(DE1,DE2)))))),((((RA2,(RA3,RA4)),RA1),RA0),(((((QD1,QD2),(QD0,(QD3,QD4))),((RD0,(RD3,RD4)),(RD1,RD2))),(ID0,((ID1,ID2),(ID3,ID4)))),((((DD1,DD2),DD0),(DD3,DD4)),(((QE0,QE1),((QE3,QE4),QE2)),(((RE3,RE4),RE2),(RE0,RE1)))))))))))),((((DB1,DB2),(DB0,(DB3,DB4))),((((EB3,EB4),EB2),EB1),EB0)),(((((HA0,HA1),(HA2,(HA3,HA4))),((((SD0,(SD1,((SD3,SD4),SD2))),(((OE1,OE2),(OE0,(OE3,OE4))),(((PE1,PE2),PE0),(PE3,PE4)))),(((BE2,(BE3,BE4)),(BE0,BE1)),((CE2,(CE3,CE4)),(CE0,CE1)))),(((((SE1,((SE3,SE4),SE2)),SE0),((TE2,(TE3,TE4)),(TE0,TE1))),(((VD1,((VD3,VD4),VD2)),VD0),((WD1,WD2),(WD0,(WD3,WD4))))),((BB1,(BB2,(BB3,BB4))),BB0)))),(((((ED1,ED2),ED0),(ED3,ED4)),((FD0,(FD3,FD4)),(FD1,FD2))),(VB0,(((VB3,VB4),VB2),VB1)))),(DA0,((DA1,DA2),(DA3,DA4))))))),(((((FC0,((FC2,(FC3,FC4)),FC1)),(GC0,((GC1,GC2),(GC3,GC4)))),(((IA3,IA4),IA2),(IA0,IA1))),(((((((YA2,(YA3,YA4)),YA1),YA0),(((ZA3,ZA4),(ZA1,ZA2)),ZA0)),(((TB2,(TB3,TB4)),(TB0,TB1)),((((JC3,JC4),JC0),(JC1,JC2)),((KC3,KC4),(KC0,(KC1,KC2)))))),(((HC0,(((HC3,HC4),HC2),HC1)),(IC0,((IC2,(IC3,IC4)),IC1))),(((RB0,(RB3,RB4)),(RB1,RB2)),((SB3,SB4),((SB1,SB2),SB0))))),(((((ZD3,ZD4),ZD2),(ZD0,ZD1)),(((AE2,(AE3,AE4)),AE1),AE0)),((EC0,(EC1,EC2)),(EC3,EC4))))),(((((NA0,(((NA3,NA4),NA2),NA1)),(((OC1,(OC2,(OC3,OC4))),OC0),((PC0,PC1),((PC3,PC4),PC2)))),(((((TD0,((TD3,TD4),(TD1,TD2))),(UD0,(UD1,(UD2,(UD3,UD4))))),(((PD1,PD2),PD0),(PD3,PD4))),((((IB3,IB4),IB2),IB1),IB0)),(((WB1,((WB3,WB4),WB2)),WB0),((((YB1,YB2),(YB0,(YB3,YB4))),(((ZB3,ZB4),ZB2),(ZB0,ZB1))),(XB0,(XB1,((XB3,XB4),XB2))))))),(((((((((UA3,UA4),UA2),UA1),UA0),((((MB1,MB2),(MB3,MB4)),MB0),(((NB1,NB2),(NB3,NB4)),NB0))),(((((XC1,XC2),((XC3,XC4),XC0)),((YC3,YC4),((YC1,YC2),YC0))),((VC0,(((VC3,VC4),VC2),VC1)),((WC1,WC2),((WC3,WC4),WC0)))),((GD0,((GD2,(GD3,GD4)),GD1)),(((HD1,HD2),(HD3,HD4)),HD0)))),(((LA0,(LA1,LA2)),(LA3,LA4)),((((VA3,VA4),VA2),(VA0,VA1)),(WA0,(((WA3,WA4),WA2),WA1))))),(((JB1,(JB2,(JB3,JB4))),JB0),(((((ZC2,(ZC3,ZC4)),ZC1),ZC0),((AD0,(AD1,AD2)),(AD3,AD4))),((((TC3,TC4),TC2),TC1),TC0)))),((BA2,(BA3,BA4)),(BA0,BA1)))),(((((GA1,GA2),GA0),(GA3,GA4)),((((((MC1,MC2),(MC0,(MC3,MC4))),((((NC3,NC4),NC2),NC1),NC0)),(((ME1,ME2),((ME3,ME4),ME0)),(((NE1,NE2),NE0),(NE3,NE4)))),((((UB3,UB4),UB2),UB1),UB0)),((KB0,(KB1,KB2)),(KB3,KB4)))),((CA1,CA2),(CA0,(CA3,CA4)))))));
