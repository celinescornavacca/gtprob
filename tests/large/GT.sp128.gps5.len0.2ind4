((((((((CB1,(CB2,(CB3,CB4))),CB0),(((ZB0,(ZB1,(ZB2,(ZB3,ZB4)))),((AC1,AC2),(AC0,(AC3,AC4)))),((GB3,GB4),((GB1,GB2),GB0)))),(((AD0,(AD1,AD2)),(AD3,AD4)),((BD3,BD4),(BD0,(BD1,BD2))))),(AA0,(AA1,(AA2,(AA3,AA4))))),((((((GD0,GD1),((GD3,GD4),GD2)),((HD1,(HD2,(HD3,HD4))),HD0)),((((ZC1,ZC2),(ZC3,ZC4)),ZC0),(((AE0,AE1),(AE2,(AE3,AE4))),((BE0,(BE3,BE4)),(BE1,BE2))))),(((((LE3,LE4),(LE0,(LE1,LE2))),(((ME1,ME2),(ME3,ME4)),ME0)),((((QA3,QA4),QA0),(QA1,QA2)),((((((KD1,(KD2,(KD3,KD4))),KD0),(((LD3,LD4),LD2),(LD0,LD1))),(((CD1,CD2),(CD0,(CD3,CD4))),(((DD3,DD4),(DD1,DD2)),DD0))),((DC1,DC2),((DC3,DC4),DC0))),((JB0,(JB3,JB4)),(JB1,JB2))))),(EA0,((EA1,EA2),(EA3,EA4))))),((((MB3,MB4),(MB1,MB2)),MB0),(((KC1,((KC3,KC4),KC2)),KC0),((LC1,LC2),((LC3,LC4),LC0)))))),(((((FD0,FD1),(FD2,(FD3,FD4))),((ZD0,(((ZD3,ZD4),ZD2),ZD1)),(((NE0,NE1),((NE3,NE4),NE2)),((OE0,(OE3,OE4)),(OE1,OE2))))),((((BB2,(BB3,BB4)),(BB0,BB1)),((((CE1,CE2),(CE0,(CE3,CE4))),(((DE2,(DE3,DE4)),DE1),DE0)),(((HB3,HB4),HB0),(HB1,HB2)))),((HA0,(((HA3,HA4),HA2),HA1)),(((((EB3,EB4),EB0),(EB1,EB2)),(FB0,(FB1,(FB2,(FB3,FB4))))),((((ID1,ID2),(ID3,ID4)),ID0),((((JD3,JD4),JD2),JD1),JD0)))))),(((((((IE0,(IE1,(IE2,(IE3,IE4)))),((UE0,((UE2,(UE3,UE4)),UE1)),((VE3,VE4),((VE1,VE2),VE0)))),(((((ZA2,(ZA3,ZA4)),ZA1),ZA0),((AB1,AB2),((AB3,AB4),AB0))),((((((ND2,(ND3,ND4)),ND1),ND0),(OD0,(OD1,((OD3,OD4),OD2)))),((QB0,((QB3,QB4),(QB1,QB2))),((XC0,(XC1,(XC2,(XC3,XC4)))),((YC3,YC4),((YC1,YC2),YC0))))),((WA1,WA2),((WA3,WA4),WA0))))),((MA1,(MA2,(MA3,MA4))),MA0)),(((GA0,GA1),(GA2,(GA3,GA4))),(((UC0,(UC1,UC2)),(UC3,UC4)),(((VC1,VC2),VC0),(VC3,VC4))))),(((XA0,(XA1,(XA2,(XA3,XA4)))),((YA0,YA1),((YA3,YA4),YA2))),(((RA1,RA2),(RA3,RA4)),RA0))),((JA0,(JA1,((JA3,JA4),JA2))),((((XD1,XD2),XD0),(XD3,XD4)),((YD2,(YD3,YD4)),(YD0,YD1))))))),(((((KB0,(KB1,((KB3,KB4),KB2))),((LB0,LB1),(LB2,(LB3,LB4)))),((((((((((RE1,RE2),(RE3,RE4)),RE0),(((SE2,(SE3,SE4)),SE1),SE0)),(SD0,((SD1,SD2),(SD3,SD4)))),(TB0,((TB3,TB4),(TB1,TB2)))),(((((SB3,SB4),SB2),SB1),SB0),(((EC0,(EC3,EC4)),(EC1,EC2)),((FC3,FC4),((FC1,FC2),FC0))))),((((JE3,JE4),JE0),(JE1,JE2)),((KE0,KE1),((KE3,KE4),KE2)))),((((PD2,(PD3,PD4)),PD1),PD0),((QD1,((QD3,QD4),QD2)),QD0))),(((((PA2,(PA3,PA4)),PA1),PA0),((((((EE2,(EE3,EE4)),EE1),EE0),((FE3,FE4),((FE1,FE2),FE0))),(WC0,((WC2,(WC3,WC4)),WC1))),((YB3,YB4),((YB1,YB2),YB0)))),(((((((TD0,TD1),((TD3,TD4),TD2)),(((UD2,(UD3,UD4)),UD1),UD0)),(PB0,(((PB3,PB4),PB2),PB1))),(DB0,(DB1,(DB2,(DB3,DB4))))),(((WB3,WB4),((WB1,WB2),WB0)),(XB0,(XB1,(XB2,(XB3,XB4)))))),((QC0,(QC1,(QC2,(QC3,QC4)))),(RC0,((RC1,RC2),(RC3,RC4)))))))),((((OC1,((OC3,OC4),OC2)),OC0),(((PC3,PC4),(PC1,PC2)),PC0)),(((((RD3,RD4),((RD1,RD2),RD0)),((((PE3,PE4),PE2),(PE0,PE1)),(QE0,((QE1,QE2),(QE3,QE4))))),(((IA0,((IA1,IA2),(IA3,IA4))),((((ED3,ED4),ED0),(ED1,ED2)),((VD0,((VD2,(VD3,VD4)),VD1)),((WD1,(WD2,(WD3,WD4))),WD0)))),(((((TA3,TA4),TA2),TA1),TA0),((GE0,((GE1,GE2),(GE3,GE4))),(HE0,(HE1,(HE2,(HE3,HE4)))))))),(((BC0,(BC3,BC4)),(BC1,BC2)),(CC0,((CC1,CC2),(CC3,CC4))))))),(((((BA1,(BA2,(BA3,BA4))),BA0),(((((((SC3,SC4),SC0),(SC1,SC2)),((TC1,TC2),((TC3,TC4),TC0))),(((CA1,CA2),CA0),(CA3,CA4))),((((UA3,UA4),(UA1,UA2)),UA0),(VA0,(((VA3,VA4),VA2),VA1)))),(((((MC3,MC4),MC0),(MC1,MC2)),((((NC3,NC4),NC2),NC1),NC0)),(((((UB3,UB4),UB2),(UB0,UB1)),(((VB2,(VB3,VB4)),VB1),VB0)),(((IB3,IB4),(IB1,IB2)),IB0))))),(((((IC1,((IC3,IC4),IC2)),IC0),((JC2,(JC3,JC4)),(JC0,JC1))),((((((MD1,MD2),(MD3,MD4)),MD0),(((WE0,(((WE3,WE4),WE2),WE1)),((XE0,XE1),((XE3,XE4),XE2))),(TE0,(((TE3,TE4),TE2),TE1)))),(((RB3,RB4),RB0),(RB1,RB2))),(((GC1,GC2),(GC0,(GC3,GC4))),(HC0,((HC3,HC4),(HC1,HC2)))))),(((KA2,(KA3,KA4)),(KA0,KA1)),((LA1,LA2),((LA3,LA4),LA0))))),((((((NA3,NA4),((NA1,NA2),NA0)),(OA0,((OA2,(OA3,OA4)),OA1))),(((FA3,FA4),FA2),(FA0,FA1))),(DA0,((DA3,DA4),(DA1,DA2)))),(((SA1,((SA3,SA4),SA2)),SA0),((((NB2,(NB3,NB4)),NB1),NB0),(OB0,(OB1,(OB2,(OB3,OB4))))))))));
