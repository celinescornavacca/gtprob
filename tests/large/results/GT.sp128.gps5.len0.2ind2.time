665.60user 3.24system 11:14.40elapsed 99%CPU (0avgtext+0avgdata 277568maxresident)k
156inputs+0outputs (0major+1167573minor)pagefaults 0swaps
