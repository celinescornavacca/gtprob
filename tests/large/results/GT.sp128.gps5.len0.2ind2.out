-1227.2267148
